DeepSpeech.ipynb

```diff
@@ -532,26 +540,28 @@
     "\n",
     "The next line of the funtion `BiRNN` does a bit more data preparation.\n",
     "```python\n",
-    "    # Split data because rnn cell needs a list of inputs for the RNN inner loop\n",
-    "    layer_3 = tf.split(0, n_steps, layer_3)\n",
+    "    # Reshape data because rnn cell needs a list of inputs for the BRNN inner loop\n",
+    "    layer_3 = tf.reshape(layer_3, [-1, batch_size, n_hidden_3])\n",
```

Comment:
Whoops, hard-coding the batch_size here - just needs to take it from the input instead, fixed locally.